pkgs/development/python-modules/yamllint/default.nix

```diff
@@ -27,8 +27,8 @@ buildPythonPackage rec {
   ];
 
   disabledTests = [
-    # Test failure could be related to https://github.com/adrienverge/yamllint/issues/334
-    "test_find_files_recursively"
+    "test_cli"
+    "test_key_ordering"
```

Comment:
Any idea why those fails? Does it give you an error message?

Reply:
For me, `test_find_files_recursively` is still the failing test. It fails because `/build` sorts before `/etc`: https://github.com/adrienverge/yamllint/blob/v1.26.0/tests/test_cli.py#L152-L158
Please file an upstream issue about that.

Reply:
@Mic92 This is an [upstream issue](https://github.com/adrienverge/yamllint/issues/307) having to do with locale on BSDs (Darwin).

I've modified my commit to wrap these in a lib.optional stdenv.isDarwin clause.

@dotlambda I've reproduced failure with test_find_files_recursively and added it back in. There seems to already be an upstream issue [here](https://github.com/adrienverge/yamllint/issues/334).